### PR TITLE
[eval] increase timeout for SWEBench eval init/complete

### PR DESCRIPTION
**Pull request by xingyaoww**
**Short description of the problem this fixes or functionality that this introduces. This may be used for the CHANGELOG**

Sometimes it takes a little while to run `git reset --hard` than the default timeout. Without a longer timeout, the runtime would return too early and causing `assert exit_code == 0` to fail.

Could fix https://github.com/All-Hands-AI/OpenHands/issues/3834

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

---
**Link of any specific issues this addresses**

### CODE REVIEW

**Review by neubig — APPROVED** [review]
Good idea!

**Comment by enyst** [comment]
I think I just hit this. Thanks for the fix!

**Comment by xingyaoww** [comment]
Actually - i think this goes further into https://github.com/All-Hands-AI/OpenHands/pull/3653 - we probably also need to change the client-side code for this. Setting this timeout was actually not honored by runtime client..

File: tests/integration/mock/eventstream_runtime/DelegatorAgent/test_edits/prompt_002.log [hunk 1]
```diff
@@ -41,7 +37,7 @@ as well as observations you've made. This only includes the MOST RECENT
 actions and observations--more may have happened before that.
 They are time-ordered, with your most recent action at the bottom.
 
-[{"source": "agent", "action": "run", "args": {"command": "ls", "thought": "", "keep_prompt": true, "is_confirmed": "confirmed"}}, {"source": "agent", "observation": "run", "content": "bad.txt\r\n\r\n[Python Interpreter: /openhands/poetry/openhands-ai-5O4_aCHf-py3.11/bin/python]\nopenhands@docker-desktop:/workspace $ ", "extras": {"command_id": -1, "command": "ls", "exit_code": 0}}]
+[{"source": "agent", "action": "run", "args": {"command": "ls", "thought": "", "blocking": false, "keep_prompt": true, "is_confirmed": "confirmed"}}, {"source": "agent", "observation": "run", "content": "bad.txt\r\n\r\n[Python Interpreter: /openhands/poetry/openhands-ai-5O4_aCHf-py3.11/bin/python]\nopenhands@xingyao-develop:/workspace $ ", "extras": {"command_id": -1, "command": "ls", "exit_code": 0}}]
```
Comment (enyst) [inline tests/integration/mock/eventstream_runtime/DelegatorAgent/test_edits/prompt_002.log:40]: This command prompt change @xingyao-develop makes me wonder how does this even work for integration tests now, don't we all have some specific prompt? Do we ignore it?

File: tests/integration/mock/eventstream_runtime/DelegatorAgent/test_write_simple_script/prompt_005.log [hunk 2]
```diff
@@ -41,7 +37,7 @@ as well as observations you've made. This only includes the MOST RECENT
...
+[{"source": "agent", "action": "write", "args": {"path": "hello.sh", "content": "#!/bin/bash\n\necho 'hello'", "start": 0, "end": -1, "thought": ""}}, {"source": "agent", "observation": "write", "content": "", "extras": {"path": "/workspace/hello.sh"}}, {"source": "agent", "action": "run", "args": {"command": "chmod +x hello.sh", "thought": "", "blocking": false, "keep_prompt": true, "is_confirmed": "confirmed"}}, {"source": "agent", "observation": "run", "content": "chmod: changing permissions of 'hello.sh': Operation not permitted\r\n\r\n[Python Interpreter: /openhands/poetry/openhands-ai-5O4_aCHf-py3.11/bin/python]\nopenhands@xingyao-develop:/workspace $ ", "extras": {"command_id": -1, "command": "chmod +x hello.sh", "exit_code": 1}}]
```
Comment (enyst) [inline tests/integration/mock/eventstream_runtime/DelegatorAgent/test_write_simple_script/prompt_005.log:40]: This had a permission error on your machine

Comment (xingyaoww) [reply tests/integration/mock/eventstream_runtime/DelegatorAgent/test_write_simple_script/prompt_005.log]: fixed!

**Comment by xingyaoww** [comment]
This should be finally ready for another review :)

**Review by neubig — APPROVED** [review]
LGTM!

**Pull request merged**